### DISCUSSION TITLE
#28. VATZ biweekly meeting at 2023-03-08 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#408] [General]
### 1. Overall
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/6308023/219247150-66cff1c7-9b59-4a55-ad58-6f9a733d61f9.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
27 | 2/19 | 10
28 | 6/25 |  24
---

<img width="922" alt="image" src="https://user-images.githubusercontent.com/6308023/222050399-f0ff420a-d53c-40c0-8a55-a383f13e325c.png">

<img width="1240" alt="image" src="https://user-images.githubusercontent.com/6308023/222050497-d8b88aa9-ed25-4f7e-8810-ffd7c75a0553.png">

<img width="929" alt="image" src="https://user-images.githubusercontent.com/6308023/223725140-45a647bd-4134-4df1-b751-37baae4bf65f.png">

**Comment by xellos00** [comment]
# 동요 

## Weekly Tasks

- #390 
- #405 
- #406 

## Security 

- #410

**Comment by rootwarp** [comment]
# 준교
N/A

**Comment by skonhwang** [comment]
## 승곤
- https://github.com/dsrvlabs/vatz/issues/410#issuecomment-1459649675

**Comment by gnongs** [comment]
## 진홍
- No issue
